src/lib.rs

```diff
@@ -596,6 +596,9 @@ pub use oauth2::{
     TokenResponse as OAuth2TokenResponse, TokenType, TokenUrl,
 };
 
+#[cfg(feature = "backchannel-logout")]
+pub mod backchannel_logout;
```

Comment:
`pub mod` seems like the right approach since OIDC supports multiple logout mechanisms and, as you pointed out, it could get confusing to put everything in the top-level namespace.

Mind moving this down to where the other pub mods (e.g.,`core`) are declared?